src/codepen.js

```diff
@@ -0,0 +1,10 @@
+import { URL } from 'url';
+
+export const shouldTransform = string => {
+  return new URL(string).host == 'codepen.io';
```

Comment:
Use an implicit return

Reply:
should i also change the file name to **CodePen.js** instead of **codepen.js**?